packages/vscode-extension/src/project/project.ts

```diff
@@ -584,7 +599,7 @@ export class Project implements Disposable, MetroDelegate, ProjectInterface {
 
   // used in callbacks, needs to be an arrow function
   private removeDeviceListener = async (_devices: DeviceInfo) => {
```

Comment:
I know it was like that but I think `_devices` should be called `device` or even better `removedDevice` since its just one.

Reply:
The underscore is signifying that the argument is not used.